packages/shared-ux/page/analytics_no_data/mocks/src/storybook.ts

```diff
@@ -34,13 +35,22 @@ export class StorybookMock extends AbstractStorybookMock<
       control: 'text',
       defaultValue: 'Kibana guide',
     },
+    customBranding: {
```

Comment:
I don't think this should be a control. I am not sure if setting boolean like this would give us the results we need.